defaults/genesis.json

```diff
@@ -149,6 +166,75 @@
           "destination": "ed0120CE7FA46C9DCE7EA4B125E2E36BDB63EA33073E7590AC92816AE1E861B7048B03@wonderland"
         }
       }
+    },
+    {
+      "Register": {
+        "Trigger": {
+          "id": "multisig_domains",
+          "action": {
+            "executable": {
+              "Wasm": "cb253c4ebe8ae68ad6591b4dd7acc447282df663fc16a3cc20f9938bfa0d3d83"
```

Comment:
didn't we say this should be path to the file?

Reply:
This was an attempt to improve readability while minimize changes except for `executable.Wasm` representation, but yeah, I'm changing approach to specify a path under `wasm_samples`. See https://github.com/hyperledger/iroha/pull/5027#discussion_r1787878806